R/downloadSource.R

```diff
@@ -73,48 +73,50 @@ downloadSource <- function(type, subtype = NULL, overwrite = FALSE) {
     if (overwrite) {
       unlink(typesubtype, recursive = TRUE)
     } else {
-      stop("Source folder for source \"", typesubtype,
-           "\" does already exist! Delete folder or activate overwrite to proceed!")
+      stop('Source folder for source "', typesubtype,
+           '" does already exist! Delete folder or activate overwrite to proceed!')
     }
   }
 
-  dir.create(typesubtype, recursive = TRUE)
-  absolutePathTypesubtype <- normalizePath(typesubtype)
-  local_dir(absolutePathTypesubtype)
+  downloadInProgressDirectory <- paste0(typesubtype, "-downloadInProgress")
```

Comment:
wouldn't the current setup mean that parallel preprocessings would now start to download the same source data in parallel into the same folder?